iroha/src/kura.rs

```diff
@@ -1,18 +1,15 @@
 //! This module contains persistence related Iroha logic.
 //! [`Kura`] is the main entity which should be used to store new [`Block`](`crate::block::VersionedCommittedBlock`)s on the blockchain.
 
-use std::{
-    fs,
-    path::{Path, PathBuf},
-    sync::Arc,
-};
+use std::{fs::{self}, path::{Path, PathBuf}, sync::Arc};
```

Comment:
Could you try not to mix async and sync api. It is bad generally

Reply:
Use tokio::fs and etc.